homeassistant/components/filter/sensor.py

```diff
@@ -454,7 +462,7 @@ def filter_state(self, new_state):
 
 
 @FILTERS.register(FILTER_NAME_RANGE)
-class RangeFilter(Filter, SensorEntity):
+class RangeFilter(Filter):
```

Comment:
I might have misunderstood something, but I see no reason for the filters to inherit `SensorEntity`

Reply:
Not that I can recall :/ 

since tests didn't break and coverage is high, I'm ok with this